R/moduleCoverage.R

```diff
@@ -47,11 +56,12 @@ setGeneric("moduleCoverage", function(name, path) {
 #' @rdname moduleCoverage
 setMethod(
   "moduleCoverage",
-  signature(name = "character", path = "character"),
-  definition = function(name, path) {
+  signature(name = "character", path = "character", byFunctionName = "logical"),
+  definition = function(name, path, byFunctionName) {
     tmpdir <- tempdir(); on.exit(unlink(tmpdir, recursive = TRUE))
-    fnDir <- file.path(path, name, "moduleFunctions") %>%
+    fnDir <- file.path(tmpdir, name, "moduleFunctions") %>%
```

Comment:
fnDir is temporary directory that hosts all the functions in module. initially, i put this directory into `path` directory. i realized that this will create a hard-coded directory. therefore, i moved fnDir into a temporary path. it has nothing to do with `path` argument. i also put outputDir and a dummy testfile into this directory.